yjit.rb

```diff
@@ -162,27 +162,6 @@ def self.runtime_stats()
     stats[:object_shape_count] = Primitive.object_shape_count
```

Comment:
If the motivation of this change was to generate the stats Hash in a single place, we probably want to move this to the Rust function as well.